tests/conftest.py

```diff
@@ -48,17 +50,16 @@ def seeded_test_client(test_client):
     :return: a flask test client with application context and seeded db.
```

Comment:
should docstring say yield instead of return?

And can we add types to signature?